lib/mods/replace-methods.js

```diff
@@ -32,6 +32,10 @@ function isPropertyOfNamespace(propertyAccess) {
   const baseModuleName = getModuleNameForNode(base);
 
   const baseSymbol = base.getSymbol();
+  if (!baseSymbol) {
```

Comment:
Would be good to document when this occurs and/or add unit tests for these isFoo helpers to make it clear what cases we are thinking about.

Reply:
The integration tests cover this case, but yeah, documenting it would be good.